source/distributed/fully_distributed_tria.cc

```diff
@@ -225,17 +225,22 @@ namespace parallel
       Assert(
         currently_processing_create_triangulation_for_internal_usage,
         ExcMessage(
-          "You have called the method parallel::fullydistributed::Triangulation::create_triangulation() \n"
-          "that takes 3 arguments. If you have not called this function directly, \n"
-          "it might have been called via a function from the GridGenerator or GridIn \n"
-          "namespace. To be able to set up a fully-distributed Triangulation with these \n"
-          "utility functions nevertheless, please follow the following three steps:\n"
-          "  1) call the utility function for a (serial) Triangulation, \n"
-          "     a parallel::shared::Triangulation, or a parallel::distributed::Triangulation object,\n"
-          "  2) use the functions TriangulationDescription::Utilities::create_description_from_triangulation() \n"
-          "     or ::create_description_from_triangulation_in_groups() to create the \n"
-          "     description of the local partition, and\n"
-          "  3) pass the created description to parallel::fullydistributed::Triangulation::create_triangulation()."));
+          "You have called the method\n"
+          "\n"
+          "    parallel::fullydistributed::Triangulation::create_triangulation()"
+          "\n"
+          "which takes 3 arguments. If you have not called this function directly,\n"
+          "it might have been called via a function from the GridGenerator or GridIn\n"
+          "namespace. To set up a fully-distributed Triangulation with these utility\n"
+          "functions, please start by using the same process to set up a serial\n"
+          "Triangulation, parallel::shared::Triangulation, or a\n"
+          "parallel::distributed::Triangulation. Once that is complete use the\n"
+          "copy_triangulation() member function to finish setting up the original fully\n"
+          "distributed Triangulation. Alternatively, you can use\n"
+          "TriangulationDescription::Utilities::create_description_from_triangulation()\n"
+          "or create_description_from_triangulation_in_groups() to create the\n"
+          "description of the local partition, and pass that description to\n"
+          "parallel::fullydistributed::Triangulation::create_triangulation()."));
```

Comment:
Actually, just remove all of the `\n` here, except in the empty lines. The exception machinery breaks long text to 80 characters.